csaf_2.1/prose/edit/src/tests-01-mndtr-51-inconsistent-epss-timestamp.md

```diff
@@ -0,0 +1,58 @@
+### Inconsistent EPSS Timestamp
+
+For each vulnerability, it MUST be tested that the each EPSS `timestamp` is earlier than or equal to the `date` of the newest item of the
+`revision_history` if the document status is `final` or `interim`.
+As the timestamps might use different timezones, the sorting MUST take timezones into account.
+
+The relevant path for this test is:
+
+```
+    /vulnerabilities[]/metrics[]/content/epss/timestamp
+```
+
+*Example 1 (which fails the test):*
```

Comment:
I can add the corresponding example entry after merge into the revision branch.

Reply:
Thank you.